scripts/dev-setup.sh

```
@@ -93,12 +101,12 @@ check_system_requirements() {
         log_success "Available memory: ${TOTAL_MEM_GB}GB ✓"
     fi
     
-    # Check disk space (minimum 10GB free)
+    # Check disk space (minimum configured)
     AVAILABLE_SPACE_KB=$(df . | tail -1 | awk '{print $4}')
     AVAILABLE_SPACE_GB=$((AVAILABLE_SPACE_KB / 1024 / 1024))
     
-    if [ $AVAILABLE_SPACE_GB -lt 10 ]; then
-        log_error "Insufficient disk space: ${AVAILABLE_SPACE_GB}GB available (minimum: 10GB)"
+    if [ $AVAILABLE_SPACE_GB -lt $MINIMUM_DISK_SPACE_GB ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)